modules/local/external_tools_import.nf

```diff
@@ -52,7 +52,7 @@ process EXTERNAL_TOOLS_IMPORT {
     # Substitution 3: NMHOME should be the folder in which the tcsh script itself resides
     #
     sed -i.bak \
-        -e 's_bin/tcsh.*\$_usr/bin/env tcsh_' \
+        -e 's_bin/tcsh.*\$_opt/conda/bin/tcsh_' \
```

Comment:
Is this changed because of the container? Shouldn't this be container-unrelated?

Reply:
Good catch, that needs to be adjusted to the paths in the new container. That one was from the Conda-based container.